src/test/java/org/example/StudentRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StudentRepositoryTest {
+
```

Comment:
add 0, 1 and 2 students, each scenario is a separate test. also test the sorting method with 2 or 3 students